nav2_behaviors/include/nav2_behaviors/timed_behavior.hpp

```diff
@@ -289,12 +290,14 @@ class TimedBehavior : public nav2_core::Behavior
   // Stop the robot with a commanded velocity
   void stopRobot()
   {
-    auto cmd_vel = std::make_unique<geometry_msgs::msg::Twist>();
-    cmd_vel->linear.x = 0.0;
-    cmd_vel->linear.y = 0.0;
-    cmd_vel->angular.z = 0.0;
-
-    vel_pub_->publish(std::move(cmd_vel));
+    auto cmd_vel = std::make_unique<geometry_msgs::msg::TwistStamped>();
+    cmd_vel->header.frame_id = "base_link";
+    cmd_vel->header.stamp = rclcpp::Time();
+    cmd_vel->twist.linear.x = 0.0;
+    cmd_vel->twist.linear.y = 0.0;
+    cmd_vel->twist.angular.z = 0.0;
+
+    // vel_pub_->publish(std::move(cmd_vel));
```

Comment:
Commented otu?

Reply:
Using std::move on the command results in a failure to compile.  It was not clear why TwistStamped needs to be a unique pointer, so I just pass it by value into publish.

Reply:
Here's a reference I found that says the usage I switched to is also valid: 
https://github.com/ros2/rclcpp/pull/676#issuecomment-489381722

What's not clear is why std::move is broken, unless the `rclcpp` maintainers changed the API again since that update.